Simulator/player.py

```diff
@@ -1,10 +1,14 @@
 import math
```

Comment:
Make sure to add Unit Tests to the code you are writing. There aren't many on the set 4 (far more than I when I started the project but less than there probably needs to be) but I want there to be plenty for set 12. Especially for Augment related behavior.